.env.sample

```diff
@@ -0,0 +1,39 @@
+#DEBUG=False
```

Comment:
Since the .env.sample file is added in this PR, maybe put an instruction line inside the README file accordingly? Something like "Copy the .env.sample file as .env and set values for those environment variables".